src/Microsoft.ApiDesignGuidelines.Analyzers/CSharp/CSharpDefineAccessorsForAttributeArguments.cs

```diff
@@ -17,9 +18,7 @@ public class CSharpDefineAccessorsForAttributeArgumentsAnalyzer : DefineAccessor
     {
         protected override bool IsAssignableTo(ITypeSymbol fromSymbol, ITypeSymbol toSymbol, Compilation compilation)
         {
-            return fromSymbol != null &&
-                toSymbol != null &&
-                ((CSharpCompilation)compilation).ClassifyConversion(fromSymbol, toSymbol).IsImplicit;
+            return fromSymbol.IsAssignableTo(toSymbol, (CSharpCompilation)compilation);
```

Comment:
We can completely avoid this file if `IsAssignableTo` was an interface method, just like the Roslyn implementation [ISemanticFactsService.IsAssignableTo](http://source.roslyn.io/#Microsoft.CodeAnalysis.Workspaces/LanguageServices/SemanticsFactsService/ISemanticFactsService.cs,91)
Then a client at language independent layer can invoke this method (see [here](http://source.roslyn.io/#Microsoft.CodeAnalysis.Features/GenerateMember/GenerateConstructor/GenerateConstructorHelpers.cs,75) for an example). 

We can probably create our own ISemanticFactsService deriving from [ILanguageService](http://source.roslyn.io/Microsoft.CodeAnalysis.Workspaces/R/cc9ae129c46089c0.html) in the core AnalyzerUtilities project and the CSharp and VB implementations of this interface can MEF export the language service. Feel free to open a separate work item for this refactoring if it is too much trouble right now.